### PR TITLE
add txlog specific item updater that allows same publisher equiv

### PR DESCRIPTION
**Pull request by AdamDM**

### CODE REVIEW

File: src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                                        dependencies.getExcludedIds()
+                                ),
+                                new FilmFilter<>(),
+                                new DummyContainerFilter<>(),
+                                new UnpublishedContentFilter<>()
+                        ))
+                )
+                .withExtractor(
+                        TopEquivalenceExtractor.create()
+                )
```
Comment (alianos-) [inline src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java]: Isn't this extractor always produce a single result? Didn't you want to allow many?

File: src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import org.atlasapi.equiv.results.extractors.TopEquivalenceExtractor;
+import org.atlasapi.equiv.results.filters.AlwaysTrueFilter;
+import org.atlasapi.equiv.results.filters.ConjunctiveFilter;
+import org.atlasapi.equiv.results.filters.DummyContainerFilter;
+import org.atlasapi.equiv.results.filters.ExclusionListFilter;
+import org.atlasapi.equiv.results.filters.FilmFilter;
+import org.atlasapi.equiv.results.filters.MediaTypeFilter;
+import org.atlasapi.equiv.results.filters.MinimumScoreFilter;
+import org.atlasapi.equiv.results.filters.PublisherFilter;
+import org.atlasapi.equiv.results.filters.SpecializationFilter;
```
Comment (alianos-) [inline src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java]: The import list might need some cleaning.

File: src/main/java/org/atlasapi/equiv/update/updaters/configuration/UpdaterConfigurationRegistry.java [hunk 3]
```diff
@@ -745,8 +746,16 @@ private static UpdaterConfiguration makeBarbTransmissionConfiguration() {
         return UpdaterConfiguration.builder()
                 .withSource(BARB_TRANSMISSIONS)
                 .withItemEquivalenceUpdater(
-                        BARB_ITEM,
-                        ImmutableSet.of(BBC_NITRO, ITV_CPS, BARB_MASTER, UKTV, C4_PMLSD, FIVE)
+                        TXLOGS_ITEM,
+                        ImmutableSet.of(
```
Comment (alianos-) [inline src/main/java/org/atlasapi/equiv/update/updaters/configuration/UpdaterConfigurationRegistry.java]: Did you mean to remove BARB_ITEM?

Comment (AdamDM) [reply src/main/java/org/atlasapi/equiv/update/updaters/configuration/UpdaterConfigurationRegistry.java]: yup

File: src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java (hunk 2 above)
Comment (AdamDM) [reply src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java]: good catch

File: src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java (hunk 1 above)
Comment (AdamDM) [reply src/main/java/org/atlasapi/equiv/update/updaters/providers/item/TxlogsItemUpdaterProvider.java]: ah ok, this might be an issue

**Pull request merged**